src/components/AlbumThumbLoading.tsx

```diff
@@ -12,11 +13,18 @@ function AlbumThumbLoading() {
           <div className="animation-date"></div>
         </div>
       </div>
-    </div>,
-    0,
+    </div>
   )
+}
 
-  return <div className="main-container">{LoadingAlbums.map((album) => album)} </div>
+function AlbumThumbLoading({ num }: { num: number }) {
+  return (
+    <div className="main-container">
```

Comment:
Don't put the container here, that should be done by the caller. Better use an empty fragment `<>...</>`.